InstallOSX.md

```diff
@@ -17,6 +17,8 @@ such as [FreeTDS](http://www.freetds.org).
 brew install freetds
 ```
 
+Note: If you install FreeTDS from another source, e.g. [MacPorts](https://www.macports.org), you might have to adjust the value for `TDS_INCLUDE` in the make calls below.
```

Comment:
Since we are now aware of the Path MacPorts uses, I'd instead write this as:

Note: If you install FreeTDS from another source, e.g. [MacPorts](https://www.macports.org), you might have to adjust the value for `TDS_INCLUDE` in the make calls below (e.g. `-I/opt/local/include/freetds` for MacPorts)